datapack.md

```diff
@@ -0,0 +1,23 @@
+# Instalacja datapacków
+Żeby wgrać na serwer datapacka, musimy go najpierw skądś pobrać. Znajdziemy je m. in. na stronach [planet minecraft](https://www.planetminecraft.com/data-packs/), [vanilla tweaks](https://vanillatweaks.net/picker/datapacks/) oraz [voodoobeard](http://mc.voodoobeard.com/).
+Następnie w plikach serwera otwieramy folder **world**, a w nim folder **datapacks**. Tam wgrywamy nasze datapacki **w formacie .zip**. Żeby to zrobić, należy nacisnąć przycisk **Plik** z zakładki **Wgraj pliki**.
+
+![1](img/datapack.png)
+
+Następnie wybieramy naszego datapacka i klikamy **otwórz**.
+
+![1](img/datapack2.png)
+
+Nasz datapack po tym powinien pojawić się w panelu serwera.
+
+![1](img/datapack3.png)
+
+Następnie włączamy serwer i wpisujemy komendę **/minecraft:reload** lub jeśli taka nie istnieje to **/reload**.

```

Comment:
Musi istnieć :P

Reply:
no chyba że lecisz na vanilli

Reply:
No ale na vanilla też musi być, bo to jest komenda z vanilla.

Reply:
Jeśli lecisz na spigocie to /reload jest od pluginów, a jeśli chcesz przeładować datapacki to piszesz /minecraft:reload. Dlatego dopisałem, że jeśli ktoś leci na vanilli i nie ma takiej komendy jak /minecraft:reload, to żeby wpisał /reload.
![image](https://user-images.githubusercontent.com/83579930/131497852-425ba34c-54ee-48ff-a594-aa10305872a9.png)
Gdy to robiłem, to odpaliłem serwer, żeby się upewnić

Reply:
A to dziwne, bo to przecież komenda z vanilla, więc `/minecraft:reload` powinno normalnie działać.